src/opening-period/form/OpeningPeriodForm.tsx

```diff
@@ -354,17 +354,19 @@ export default function OpeningPeriodForm({
           </p>
         </div>
         <div className="opening-period-final-action-row-container">
```

Comment:
className="opening-period-form-action-row"?

Reply:
There's already a class named form-actions-row, so it would probably semantically overlap with it 🤔